site/src/sphinx/server-annotated-service.rst

```diff
@@ -439,7 +439,8 @@ converter is not able to convert the request.
     }
 
 Then, you can write your service method as follows. Note that a request converter will work on the parameters
-which are annotated with :api:`@RequestObject`.
+which are annotated with :api:`@RequestObject`. You can omit the annotation in the parameter list
+of your service method.
```

Comment:
How about omitting the 'Note that...' sentence completely so our users avoid using `@RequestObject` where unnecessary?